server/src/pages/homepage/view.js

```diff
@@ -167,7 +167,7 @@ Body.propTypes = {
   hasFxa: PropTypes.bool,
   firefoxVersion: PropTypes.string,
   isFirefox: PropTypes.bool,
-  showMyShots: PropTypes.bool,
+  authenticated: PropTypes.bool,
```

Comment:
Removing showMyShots that was previously checking for deviceId in favor of authenticated set inside ReactRender.js when a deviceId or accountId auth is found